packages/material-ui/src/styles/createPalette.js

```diff
@@ -61,8 +61,8 @@ export const dark = {
   },
   divider: 'rgba(255, 255, 255, 0.12)',
   background: {
-    paper: grey[800],
-    default: '#303030',
+    paper: '#121212',
+    default: '#121212',
```

Comment:
Looking into the usage of background level1 and 2 I see it is for hard-coding elevation into the documentation's appBar, drawer etc. 
If the color by elevation logic will be implemented the reason for having background levels is not needed because the paper in those components will provide the proper color. 
I think providing the elevation colors built in, as a default option at least, will be very comfortable in most use cases.

Reply:
> Looking into the usage of background level1 and 2

We aim to remove the custom theme values of the documentation. It's a signal that the default palette is wrong.

We definitely want to fix #18309